### DISCUSSION TITLE
Lua Tutorials

### DISCUSSION THREAD

**Discussion by Antix-Development** [o3de/o3de#6153] [General]
Hi, are there any tutorials using Lua? On the YouTube channel I can only see the script canvas stuff.

**Comment by Zaya6** [comment]
There are a few small ones on the wiki of this project https://github.com/o3de/o3de/wiki/%5BLua%5D-Hello-World
Looks like there's much more still to do with cleaning and building documentation

**Reply by Antix-Development** [reply]
Thanks. Yep I found those now also. Looks like its early days still.

**Reply by Zaya6** [reply]
> Thanks. Yep I found those now also. Looks like its early days still.

Hey I just found some more over here if you're still looking https://o3de.org/docs/user-guide/scripting/lua/

**Reply by Antix-Development** [reply]
Awesome, thanks :)
